barretenberg/cpp/src/barretenberg/vm/avm/generated/full_row.cpp

```diff
@@ -203,7 +203,6 @@ template <typename FF> std::vector<std::string> AvmFullRow<FF>::names()
              "main_sel_op_cast",
              "main_sel_op_chain_id",
              "main_sel_op_cmov",
-             "main_sel_op_coinbase",
```

Comment:
Some kind of weird base diff I'm assuming?

Reply:
yeah don't worry about AVM diff, this is generated code